addons/web/static/src/legacy/js/views/basic/basic_model.js

```diff
@@ -3992,6 +3992,11 @@ var BasicModel = AbstractModel.extend({
                     }
                     return ids;
                 }
+                if (field.type === "properties" && _.isArray(fieldValue)) {
+                    // remove deleted properties to be able
+                    // to filter based on empty properties field
+                    return fieldValue.filter(definition => !definition.definition_deleted);
```

Comment:
How does it behave if you compare the properties value to "False" in the domain? Should that be supported?

For regular fields we usually compare to False, meaning "it's not set" when building domains.

Reply:
Here we can never read False on a properties field, when we generate it (by merging with the definition record) we always generate a list

The comparison stuff is done in `web/static/src/core/domain.js@matchCondition`, I can add a condition to support that case, but this function is light so I would prefer to keep as simple as possible (and to do just what we need to remove the compute in knowledge)